package/bin/splunk_ta_modinput_rh_perform_crd_operation.py

```diff
@@ -33,7 +33,7 @@ def read_file(self, confInfo):
         file_path = self.callerArgs.data['file_path'][0]
 
         if os.path.exists(file_path):
-            with open(file_path, 'r') as ckpt_file:
+            with open(file_path, 'rb') as ckpt_file:
```

Comment:
I think with this change, now the bytes must be encoded before adding to the confInfo dictionary.